src/profile-logic/profile-data.js

```diff
@@ -1189,6 +1189,10 @@ export function filterTracingMarkersToRange(
   );
 }
 
+export function isNetworkMarker(marker: TracingMarker): boolean {
+  return !!(marker.data && marker.data.type === 'Network');
```

Comment:
I'm pretty sure the !! is unnecessary.

Reply:
Flow wants it because otherwise when `marker.data` is null it would return null, which isn't a boolean.

Alternatives are:
* returning `boolean | null`
* use a ternary or a `if` block to always return a boolean.

I prefer my solution but tell me your preference :)

Reply:
You're right, I was confused about what && returns if the first argument is falsy.

Reply:
Let's keep what you have.